source/_integrations/mawaqit.markdown

```
@@ -0,0 +1,47 @@
+
+## Prerequisites
+- a MAWAQIT.net account
+
+{% include integrations/config_flow.md %}
+
+## Integration Sensors
+
+The following sensors are added by the integration:
+
+sensors:
+  - fajr: Show the fajr prayer time for today.
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)